actionview/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Transforms a Hash into HTML Attributes, ready to be interpolated into ERB.
```

Comment:
This is a bit confusing. Maybe "Adds `tag.attributes`, which transforms a Hash into HTML Attributes, ready to be interpolated into ERB." ?